tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Builders/ServiceRecipientBuilder.cs

```diff
@@ -0,0 +1,34 @@
+﻿using NHSD.BuyingCatalogue.Ordering.Domain;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Builders
+{
+    internal sealed class ServiceRecipientBuilder
+    {
+        private readonly ServiceRecipient _serviceRecipient;
+
+        private ServiceRecipientBuilder()
+        {
+            _serviceRecipient = new ServiceRecipient
+            {
+                Name = "Some name",
+                Order = new Order()

```

Comment:
Would like to suggest the use of the OrderBuilder class here rather than creating the Order class directly. If the Order constructor changed we just want that to impact one area of the tests, not multiple.

Also the other builder pattern might be a better fit here. Specify each property of the `ServiceRecipient` separately and create the object within the `Build` method. Might be best speaking about this one & seeing what you think but use the `AddressBuilder` as an example.